src/packages/@ncigdc/modern_components/ClinicalAnalysis/ContinuousAggregationQuery.js

```diff
@@ -207,51 +228,28 @@ const getContinuousAggs = ({
 export default compose(
   withState('aggData', 'setAggData', null),
   withState('isLoading', 'setIsLoading', 'first time'),
-  withProps({
-    updateData: async ({
-      fieldName,
-      filters,
-      hits,
-      setAggData,
-      setIsLoading,
-      stats,
-      variable,
+  withPropsOnChange((props, nextProps) => 
```

Comment:
just a formatting change, move `(props, nextProps) =>` to the next line to prevent weird indentation